packages/utilities/psammead-locales/README.md

```diff
@@ -16,13 +17,22 @@ npm install @bbc/psammead-locales --save
 
 ## Usage
 
+### Moment locales
+
 ```jsx
 import moment from 'moment';
 import '@bbc/psammead-locales/moment/ig';
 
 moment.locale('ig');
 ```
 
+### Numerals
+
+```jsx
+import { Nepali } from '@bbc/psammead-locales/numerals';
```

Comment:
This made me wonder, would we actually be importing these one-by-one in practice? Or is there going to be some kind of general mapping from service/script to the correct right of numerals, indexed by name?

Reply:
Maybe we should add standard latin numbers too, so each service can have one of these in their config, and used the same across the board, rather than some having them as edge cases

Reply:
@jamesdonoh There will be a mapping. However, it will be carried out per-component. For example, for the Arabic and Urdu services want Western Arabic numerals (1,2,3,...) whereas Persian and Pashto want Eastern Arabic numerals (۱,۲,۳,...).
@dr3 The 'latin' numerals are available under WesternArabic in this package.

Reply:
@sareh ah so the same service might use different numerals for different components? Makes sense then - thanks for the explanation.

Reply:
After reviewing with @laurapsn, she stated that Eastern Arabic numerals will only be used for Most Read for Persian and Pashto services, Bengali and Burmese would be used for their respective services & Western Arabic numerals for all other services.